src/System.Security.Cryptography.Pkcs/Directory.Build.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.4.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
+    <NetstandardAssemblyVersion>4.0.4.0</NetstandardAssemblyVersion>
```

Comment:
You could have done `<AssemblyVersion Condition="'$(TargetsNetStandard)' == 'true'">4.0.4.0</AssemblyVersion>` here, depending on what makes the most sense to you.

Reply:
Consider adding a comment here that says something like netstandard API surface is still 4.0.* since that is all that is supported by desktop.

Reply:
Ah, okay, I'll try that, since it reduces the places that can get it wrong.